packages/components/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 
 -   `Modal`: Modal dialog small improvement for elementShouldBeHidden ([#65941](https://github.com/WordPress/gutenberg/pull/65941)).
 -   `Tabs`: revamped vertical orientation styles ([#65387](https://github.com/WordPress/gutenberg/pull/65387)).
+-   `ItemGroup`: default `isBordered` and `isSeparated` to `true` ([#66164](https://github.com/WordPress/gutenberg/pull/66164)).
```

Comment:
I'm not sure I understand why is this change needed — couldn't we just set the two props to `true` in `PaletteEdit`?
(also, technically this is a breaking change and we'd normally try to avoid that as much as possible)

Reply:
As described in the PR, I'm fine with a breaking change for a few reasons:
- The sole usage of this component is in `PaletteEdit`
- The default non-bordered behavior of the component is awkward
- The component is experimental, so it's perfectly OK to set better defaults, even through a breaking change. 

Related to the CHANGELOG entry, if this is a breaking change to an experimental API, do we treat it as a breaking change? It wasn't an official public API, after all. Happy to move it to "Breaking Changes", but to me it makes sense if that's reserved to changes to components that aren't private, experimental or unstable.

Reply:
> The sole usage of this component is in `PaletteEdit`

That's definitely a point to keep in mind when judging the impact of these changes — although I haven't looked on third-party usage.

> The default non-bordered behavior of the component is awkward

That's absolutely true. In general, there's a lot of [improvement work on `ItemGroup`](https://github.com/wordPress/gutenberg/issues/64445) on the horizon

> The component is experimental, so it's perfectly OK to set better defaults, even through a breaking change.

While the component is marked as experimental, it is _de facto_ considered stable by some folks given that it was included in a WordPress release — hence why I questioned this change in the first place.

Having said that, we can definitely introduce this change if we think it improves the component — we should provide a dev note to let folks know (with an example of how to revert to previous styling, though).

> Related to the CHANGELOG entry, if this is a breaking change to an experimental API, do we treat it as a breaking change? It wasn't an official public API, after all. Happy to move it to "Breaking Changes", but to me it makes sense if that's reserved to changes to components that aren't private, experimental or unstable.

I'm tempted to mark it as a Breaking Change, since I believe that recently we've been treating the "Experimental" section of the changelog as referred to the truly experimental (ie. exposed via private APIs) components. But I don't have a strong opinion — @mirka , do you have any thoughts?

Reply:
> That's definitely a point to keep in mind when judging the impact of these changes — although I haven't looked on third-party usage.

We should be good: https://wpdirectory.net/search/01JAAG2CQV99VX0PZX2PER2BFA

> That's absolutely true. In general, there's a lot of improvement work on ItemGroup on the horizon

Yep, thanks for confirming, I actually did link that in the PR description already. 

> While the component is marked as experimental, it is de facto considered stable by some folks given that it was included in a WordPress release — hence why I questioned this change in the first place.

Given that the name hints it is experimental, and docs are clear, I think the limitations here are self-imposed, and we need to be more flexible. Given that I don't see any other usage of that component, I'm confident to make this change at this time.

> Having said that, we can definitely introduce this change if we think it improves the component — we should provide a dev note to let folks know (with an example of how to revert to previous styling, though).

Good point, happy to provide a dev note if we end up landing it.

> I'm tempted to mark it as a Breaking Change, since I believe that recently we've been treating the "Experimental" section of the changelog as referred to the truly experimental (ie. exposed via private APIs) components. But I don't have a strong opinion — @mirka , do you have any thoughts?

Happy to be flexible with whatever the consensus is - let me know.